README.md

```diff
@@ -1,70 +1,105 @@
-# Getting Started with Create React App
 
```

Comment:
The video link in the PR description has no content and the video is just a few seconds.

The content of the video presentation of the project should include:
  a description of the project.
  a demo of the project features.
  you should also highlight some interesting piece of code or something you built that you are very proud of.
  Please record a new video.

Reply:
here is the [video](https://www.loom.com/share/9bd7dc065ee8469e9f89044dfa0007d1)